src/React.js

```diff
@@ -47,7 +46,9 @@ function createClass(baseClass) {
         baseClass.call(this, props);
         var spec = ctrFn(this)();
         for (var k in spec) {
-          bindProperty(this, k, spec[k]);
+          if (Object.prototype.hasOwnProperty.call(spec, k)) {
```

Comment:
This relates to the eslint rule about guarding for / in to ensure the key exists:

https://eslint.org/docs/rules/guard-for-in